docs/cloud/pouta/vm-flavors-and-billing.md

```diff
@@ -314,8 +314,9 @@ performance is significantly better.
 
 **io.70GB-700GB:**
 
-NOTE! These servers have non-redundant disks, and you may expect data loss in case of disk failure.
-NOTE! These virtual machines can not be migrated nor resized.
+!!! Note  
```

Comment:
If you wish - there are fancy new note-types available: https://docs.csc.fi/ref/#styles-available-with-type-qualifiers

Reply:
I think a `!!! Note` like I did is enough but thank you for the link 🙂